scripts/irods/test/test_iticket.py

```diff
@@ -766,3 +766,47 @@ def test_write_tickets_for_collections_allow_creation_of_data_objects__issue_591
                 self.admin.run_icommand(['ichmod', '-M', 'own', self.admin.username, fname])
 
             self.admin.run_icommand(['iticket', 'delete', ticket_name])
+
+    def test_ticket_create_with_relative_paths__issue_7103(self):
+        collection_1, collection_1_ticket_name = 'issue_7103_collection_1', 'collection_1_ticket'
+        collection_2, collection_2_ticket_name = 'issue_7103_collection_2', 'collection_2_ticket'
+        data_object_1, data_object_1_ticket_name = 'issue_7103_data_object_1', 'data_object_1_ticket'
+        data_object_2, data_object_2_ticket_name = 'issue_7103_data_object_2', 'data_object_2_ticket'
+        home_data_object, home_data_object_ticket_name = 'issue_7103_home_data_object', 'home_data_object_ticket'
+
+        home_collection_ticket_name = 'home_collection_ticket'
+
+        # Create collections
+        self.admin.assert_icommand(['imkdir', collection_1])
+        self.admin.assert_icommand(['imkdir', collection_2])
+
+        # Create objects
+        self.admin.assert_icommand(['itouch', f'{collection_1}/{data_object_1}'])
+        self.admin.assert_icommand(['itouch', f'{collection_2}/{data_object_2}'])
+        self.admin.assert_icommand(['itouch', home_data_object])
+
+        self.admin.assert_icommand(['icd', collection_1])
+
+        # Create ticket for current working directory
+        self.admin.assert_icommand(['iticket', 'create', 'read', '.', collection_1_ticket_name])
+        self.admin.assert_icommand(['iticket', 'ls', collection_1_ticket_name], 'STDOUT', [collection_1_ticket_name])
```

Comment:
Should a different user use the tickets in this test? Should we run the tests as a rodsuser?

Reply:
Okay. Admin-only, full-permissions tests have bitten us in the past which is why I bring it up.